apps/explorer/config-default.yaml

```diff
@@ -0,0 +1,317 @@
+  config:
+    - networkId: 1
+      blockNumber: 9340147
+    - networkId: 4
+      blockNumber: 5844678
+    - networkId: 100
+      blockNumber: 11948310
+
+# Wallet Connect
+walletConnect:
+  bridge: 'wss://safe-walletconnect.gnosis.io/'
+
+# tokens that aren't allowed to be traded nor deposited
+disabledTokens:
+  1: # Mainnet
+    - address: '0x57Ab1E02fEE23774580C119740129eAC7081e9D3'
+      name: Synth sUSD (deprecated)
+      symbol: sUSD-old
+      description: This token is disabled for trading and depositing. sUSD will be deprecated and replaced by another token at the end of July 2020.  Go to https://www.synthetix.io for more information
+      reason: DEPRECATED
+      url: https://docs.synthetix.io/integrations/guide/#proxy-deprecation
+
+    - address: '0xC011A72400E58ecD99Ee497CF89E3775d4bd732F'
+      name: Synthetix Network Token (deprecated)
+      symbol: SNX-old
+      description: This token is disabled for trading and depositing. SNX will be deprecated and replaced by another token at the end of July 2020.  Go to https://www.synthetix.io for more information
+      reason: DEPRECATED
+      url: https://docs.synthetix.io/integrations/guide/#proxy-deprecation
+
+    - address: '0x45804880De22913dAFE09f4980848ECE6EcbAf78'
+      name: Paxos Gold
+      symbol: PAXG
+
+    - address: '0xa7DE087329BFcda5639247F96140f9DAbe3DeED1'
+      name: Statera
+      symbol: STA
+
+      # aTokens:
+    - address: '0x7D2D3688Df45Ce7C552E19c27e007673da9204B8'
+    - address: '0xA64BD6C70Cb9051F6A9ba1F163Fdc07E0DfB5F84'
+    - address: '0x71fc860F7D3A592A4a98740e39dB31d25db65ae8'
+    - address: '0x9bA00D6856a4eDF4665BcA2C2309936572473B7E'
+    - address: '0x3a3A65aAb0dd2A17E3F1947bA16138cd37d08c04'
+    - address: '0xfC1E690f61EFd961294b3e1Ce3313fBD8aa4f85d'
+    - address: '0x9D91BE44C06d373a8a226E1f3b146956083803eB'
+    - address: '0x4DA9b813057D04BAef4e5800E36083717b4a0341'
+    - address: '0x328C4c80BC7aCa0834Db37e6600A6c49E12Da4DE'
+    - address: '0x7deB5e830be29F91E298ba5FF1356BB7f8146998'
+    - address: '0xE1BA0FB44CCb0D11b80F92f4f8Ed94CA3fF51D00'
+    - address: '0x6ee0f7bb50a54ab5253da0667b0dc2ee526c30a8'
+    - address: '0x625aE63000f46200499120B906716420bd059240'
+    - address: '0x6FCE4A401B6B80ACe52baAefE4421Bd188e76F6f'
+    - address: '0x71010A9D003445aC60C4e6A7017c1E89A477B438'
+    - address: '0x6Fb0855c404E09c47C3fBCA25f08d4E41f9F062f'
+    - address: '0xFC4B8ED459e00e5400be803A9BB3954234FD50e3'
+  4: [] # Rinkeby
+  100: [] # xDAI
+
+# Initial token selection for the sell token and buy token
+initialTokenSelection:
+  # defaults
+  sellToken: DAI
+  receiveToken: USDC
+  # network override - selected within component as reliant on network
+  networks:
+    '100':
+      sellToken: wxDAI
+      receiveToken: USDC
+
+# Initial Token List
+#   List of token loaded by default
+#   This list is used while the app loads the TCR (if the TCR setting is used)
+initialTokenList:
```

Comment:
None of this is needed

Reply:
Addressed here: https://github.com/cowprotocol/cowswap/pull/3593